zh/get-started.md

```diff
@@ -1,6 +1,403 @@
+    ```
+
+    </div>
+
+    <div label="TiKV">
+
+    创建 TiKV 组件：
+
+    ```yaml
+    apiVersion: core.pingcap.com/v1alpha1
+    kind: TiKVGroup
+    metadata:
+      name: tikv
+      namespace: db
+    spec:
+      cluster:
+        name: basic
+      replicas: 1
+      template:
+        metadata:
+          annotations:
+            author: pingcap
+        spec:
+          version: v8.1.0
+          volumes:
+          - name: data
+            mounts:
+            - type: data
+            storage: 100Gi
+    ```
+
+    ```shell
+    kubectl apply -f tikv.yaml --server-side
+    ```
+
+    </div>
+
+    <div label="TiDB">
+
+    创建 TiDB 组件：
+
+    ```yaml
+    apiVersion: core.pingcap.com/v1alpha1
+    kind: TiDBGroup
+    metadata:
+      name: tidb
+      namespace: db
+    spec:
+      cluster:
+        name: basic
+      replicas: 1
+      template:
+        metadata:
+          annotations:
+            author: pingcap
+        spec:
+          version: v8.1.0
+    ```
+
+    ```shell
+    kubectl apply -f tidb.yaml --server-side
+    ```
+
+    </div>
+    </SimpleTab>
+
+    方法二：将以上 YAML 文件保存在本地目录中，并使用以下命令一次性部署 TiDB 集群
+
+    ```shell
+    kubectl apply -f ./<directory> --server-side
+    ```
+
+3. 查看 Pod 状态
+
+    ``` shell
+    watch kubectl get pods -n db
+    ```
+
+    <details>
+    <summary>点击查看期望输出</summary>
+
+    ```
+    NAME               READY   STATUS    RESTARTS   AGE
+    pd-pd-68t96d       1/1     Running   0          2m
+    tidb-tidb-coqwpi   1/1     Running   0          2m
+    tikv-tikv-sdoxy4   1/1     Running   0          2m
+    ```
+
+    </details>
+
+    所有组件的 Pod 都启动后，每种类型组件（`pd`、`tikv` 和 `tidb`）都会处于 Running 状态。此时，你可以按 <kbd>Ctrl</kbd>+<kbd>C</kbd> 返回命令行，然后进行下一步。
+
+## 第 4 步：连接 TiDB 集群
+
+由于 TiDB 支持 MySQL 传输协议及其绝大多数的语法，因此你可以直接使用 `mysql` 命令行工具连接 TiDB 进行操作。以下说明连接 TiDB 集群的步骤。
+
+### 安装 `mysql` 命令行工具
+
+要连接到 TiDB，你需要在使用 `kubectl` 的主机上安装与 MySQL 兼容的命令行客户端。可以安装 MySQL Server、MariaDB Server 和 Percona Server 的 MySQL 可执行文件，也可以从操作系统软件仓库中安装。
+
+### 转发 TiDB 服务 4000 端口
+
+本步骤将端口从本地主机转发到 Kubernetes 中的 TiDB **Service**。
+
+首先，获取 tidb-cluster 命名空间中的服务列表：
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+kubectl get svc -n db
+```
+
+<details>
+<summary>点击查看期望输出</summary>
+
+```
+NAME             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)               AGE
+pd-pd            ClusterIP   10.96.229.12    <none>        2379/TCP,2380/TCP     3m
+pd-pd-peer       ClusterIP   None            <none>        2379/TCP,2380/TCP     3m
+tidb-tidb        ClusterIP   10.96.174.237   <none>        4000/TCP,10080/TCP    3m
+tidb-tidb-peer   ClusterIP   None            <none>        10080/TCP             3m
+tikv-tikv-peer   ClusterIP   None            <none>        20160/TCP,20180/TCP   3m
+```
+
+</details>
+
+这个例子中，TiDB **Service** 是 **tidb-tidb**。
+
+然后，使用以下命令转发本地端口到集群：
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+kubectl port-forward -n db svc/tidb-tidb 14000:4000 > pf14000.out &
+```
```

Comment:
I think if we are using MacOS, we need to config extr port mappings when create the kind cluster: https://kind.sigs.k8s.io/docs/user/configuration/#extra-port-mappings